### ISSUE TITLE
More docs about setting up a new pipeline

### ISSUE THREAD

**Issue by ewels** [nf-core/nf-co.re#42]
Add some docs about how we work with test data, docker hub, Travis etc. Maybe a walkthrough of the steps involved in adding a new pipeline.

x-ref: https://github.com/nf-core/test-datasets/pull/13

**Comment by drpatelh** [comment]
This would be great @ewels ! I used this blog to work out how to do the branching for the test dataset (https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/). Quite good. The only thing I couldnt figure out was how to change the branch to the relevant name when creating the pull request. (from docs - "Make a PR on a new branch with a relevant name"). It was quite late so I cheated and created it against the master branch. Oops.
